src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetCohorts/GetCohortsHandler.cs

```diff
@@ -51,7 +51,7 @@ public async Task<GetCohortsResult> Handle(GetCohortsQuery command, Cancellation
                 var cohortIds = cohortFiltered.Select(x => x.CohortId);
 
                 //Get apprentices count for cohorts.
-                var apprenticesTask = _db.Value.DraftApprenticeships.Where(y => cohortIds.Contains(y.CommitmentId))
+                var apprentices = await _db.Value.DraftApprenticeships.Where(y => cohortIds.Contains(y.CommitmentId))
```

Comment:
any reason for awaiting these 2 queries immediately now?

Reply:
Yep. EF Core  3 doesn't like multiple queries executing at the same time.

Reply:
you are very right